### DISCUSSION TITLE
v1.2.2

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/pogoplusle#29] [Announcements]
Fixes.

**Full Changelog**: https://github.com/Mygod/pogoplusle/compare/v1.2.1...v1.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/pogoplusle/releases/tag/v1.2.2'>v1.2.2</a>.</em>

**Comment by JuanPuch** [comment]
Hey i was trying to connect mi TECNO 10 pro but it just keep stuck at paiting dialog. I tried restarting my phone, it worked once and then stopped working again. Can you give me some advice please? I would appreciate it a lot. Thank you for made this app! It's just great. Its working 100% on my Xiaomi 11T pro. Edit: I already saw a post from July 19, i followed all steps but i´m unable to connect.  :(

**Reply by Mygod** [reply]
Maybe you have other accessibility services enabled that might cause the issue?
